src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -243,14 +243,21 @@ const JobHistory = _.flow(
               }
             },
             {
-              size: { basis: 200 },
+              size: { basis: 150, grow: 0 },
               headerRenderer: () => h(HeaderCell, ['Submitted']),
               cellRenderer: ({ rowIndex }) => {
                 const { submissionDate } = submissions[rowIndex]
                 return h(TooltipTrigger, { content: Utils.makeCompleteDate(submissionDate) }, [
                   h(TextCell, Utils.makePrettyDate(submissionDate))
                 ])
               }
+            },
+            {
+              headerRenderer: () => h(HeaderCell, ['Data entity']),
+              cellRenderer: ({ rowIndex }) => {
+                const { submissionEntity: { entityName, entityType } } = submissions[rowIndex]
```

Comment:
```suggestion
                const { submissionEntity: { entityName, entityType } = {} } = submissions[rowIndex]
```
`submissionEntity` will be `undefined` if data model was not used.